packages/devtools_app/lib/src/screens/inspector_v2/inspector_controller.dart

```diff
@@ -714,12 +625,9 @@ class InspectorController extends DisposableController
 
     lastExpanded = null; // New selected node takes precedence.
     endShowNode();
-    final detailsLocal = details;
-    final parantLocal = parent;
-    if (detailsLocal != null) {
-      detailsLocal.endShowNode();
-    } else if (parantLocal != null) {
-      parantLocal.endShowNode();
+    final parentLocal = parent;
+    if (parentLocal != null) {
+      parentLocal.endShowNode();
     }
```

Comment:
this can all be written as
`parent?.endShowNode()`

Reply:
Turns out we can remove `parent` all together :)